test/test_helper.rb

```diff
@@ -64,3 +64,12 @@ def setup
       instrumentation_service: @service
   end
 end
+
+# Provide information about socket creation
+class SocketHelpers
+  class << self
+    def default_socket_opts
+      { connect_timeout: Net::LDAP::Connection::DefaultConnectTimeout }
+    end
+  end
+end
```

Comment:
This seems overkill. Could you inline this to the tests? My testing preferences are in https://github.com/jch/styleguide/blob/master/rails.md#tests